src/analysis/normed_space/basic.lean

```diff
@@ -914,6 +959,21 @@ instance normed_algebra.id (𝕜 : Type*) [normed_field 𝕜] : normed_algebra 
 { norm_algebra_map_eq := by simp,
 .. algebra.id 𝕜}
 
+@[simp] lemma normed_algebra.norm_one (𝕜 : Type*) (𝕜' : Type*) [normed_field 𝕜]
+  [normed_ring 𝕜'] [h : normed_algebra 𝕜 𝕜'] : ∥(1:𝕜')∥ = 1 :=
+by simpa using (norm_algebra_map_eq 𝕜' (1:𝕜))
+
+lemma normed_algebra.zero_ne_one (𝕜 : Type*) (𝕜' : Type*) [normed_field 𝕜]
+  [normed_ring 𝕜'] [h : normed_algebra 𝕜 𝕜'] : (0:𝕜') ≠ 1 :=
+begin
+  refine (norm_pos_iff.mp _).symm,
+  rw normed_algebra.norm_one 𝕜 𝕜', norm_num,
+end
+
+lemma normed_algebra.to_nonzero {𝕜 : Type*} (𝕜' : Type*) [normed_field 𝕜]
```

Comment:
Why do you have an implicit `𝕜` here and an explicit `𝕜` in `normed_algebra.zero_ne_one`?

Reply:
resolved at [Zulip](https://leanprover.zulipchat.com/#narrow/stream/144837-PR-reviews/topic/.233210.2C.20openness.20of.20units)